src/client/app/components/unit/CreateUnitModalComponent.tsx

```diff
@@ -69,11 +69,13 @@ export default function CreateUnitModalComponent() {
 	/* Create Unit Validation:
 		Name cannot be blank
 		Sec in Rate must be greater than zero
+		If type of unit is suffix their must be a suffix
 	*/
 	const [validUnit, setValidUnit] = useState(false);
 	useEffect(() => {
-		setValidUnit(state.name !== '' && state.secInRate > 0);
-	}, [state.name, state.secInRate]);
+		setValidUnit(state.name !== '' && state.secInRate > 0 &&
+			(state.typeOfUnit !== UnitType.suffix || (state.typeOfUnit === UnitType.suffix && state.suffix !== '')));
```

Comment:
The first check to make sure it is unequal suffix does not seem to be needed. It also does not follow the other checks in this file. I changed it and did not see any issue. Am I missing something?

This also applies to edit.

Reply:
It's redundant, I believe i did that early on because I was looking at this line in the CreateMeter file
             (state.area === 0 || (state.area > 0 && state.areaUnit !== AreaUnitType.none
Which is unnecessary in this case.